realm/src/main/java/io/realm/Realm.java

```diff
@@ -946,6 +946,30 @@ boolean contains(Class<? extends RealmObject> clazz) {
     }
 
     /**
+     * Return a distinct set of objects of a specific class. If no objects exist, the
```

Comment:
Also describe the policy for which element is returned for the distinct set (first, last, undefined)?